recipes/editline/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "editline" %}
```

Comment:
Should the recipe be called `libeditline`?

Reply:
No. The project is called editline.